Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml

```diff
@@ -53,3 +53,20 @@
         - id: GunSafeEnergyGunMini
         - id: GunSafePistolUniversal
         - id: GunSafeSubMachineGunWt550
+
+- type: entity
+  parent: BaseGunSafeSpawner
+  id: SuperweaponSafeSpawner
+  suffix: Superweapon
+  components:
+  - type: EntityTableSpawner
+    table: !type:NestedSelector
+      tableId: RandomSuperweaponSafeTable
```

Comment:
can't you just do a groupselector with `GunSafeRocketLauncher` and `GunSafeBeamDevastator` as children like above?